ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__arm64-nightly-4.19-upgrade-from-stable-4.17.yaml

```diff
@@ -0,0 +1,92 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.19"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tools:
+    name: "4.17"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  arm64-latest:
+    release:
+      architecture: arm64
+      channel: fast
+      version: "4.17"
+  arm64-target:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.19"
+  custom:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.18"
+  intermediate:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.18"
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.17"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: baremetal-upi-ovn-ipv4-f28
+  cluster: build05
+  cron: 33 15 30 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      architecture: arm64
+      masters: "3"
+      workers: "2"
+    test:
+    - ref: cucushift-upgrade-setedge-2hops
```

Comment:
The same issue as in 4.18 https://github.com/openshift/release/pull/58626#discussion_r1835857478

Reply:
Thanks, I will submit a commit once rehearse jobs finished

Reply:
@jianlinliu @liangxia @jiajliu Do we have any change on arm arch? Following job is using `cucushift-upgrade-setedge-2hops` and seems work fine:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/59758/rehearse-59758-periodic-ci-openshift-openshift-tests-private-release-4.19-arm64-nightly-4.19-upgrade-from-stable-4.17-baremetal-upi-ovn-ipv4-f28/1867028618853486592

Reply:
There is no change. 

The reason of your job get passed is that, though `cucushift-upgrade-setedge-2hops` is setting a wrong arch of target payload (expecting an arm arch payload), `cucushift-upgrade-drop-last-hop` in `openshift-upgrade-qe-test-chainupgrade` chain will drop the wrong target payload edge, so the chain upgrade workaround the issue.

While if we run stable -> stable chain upgrade for arm ci,  the wrong target payload setting would bring some problems.

The cleanest and ideal solution is something like https://github.com/openshift/release/pull/58433#discussion_r1827310040

Reply:
oh, I see, the target version (final version) in cucushift-upgrade-setedge-2hops is not correct. I will use the original way to run arm jobs.
Thanks @jianlinliu